index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Leticia Tran</title>
+  <link href="styles/normalize.css" rel="stylesheet" />
+  <link href="styles/styles.css" rel="stylesheet" />
+</head>
+<body>
+  <!-- TODO: Have the option to listen to a playlist or at least just one song...? -->
+
+  <section class="top-bottom nav-bar">
+    <nav>
+      <ul>
+        <!-- TODO: maybe have a nav buttom thatscrolls down to the page portion with a certain section? instead of taking you to another page  -->
+        <li><a href="index.html" id="link-selected">Home</a></li>
+        <li><a href="pages/about.html">About Me</a></li>
+        <!-- TODO: more detailed stuff... -->
+        <li><a href="pages/works.html">Works</a></li>
+        <li><a href="pages/social.html">Social Media</a></li>
+        <!-- <li>Blog?</li> -->
+        <li><a href="pages/contact.html">Contact</a></li>
+
+      </ul>
+    </nav>
+  </section>
+
+  <section class="intro">
+    <!-- TODO: put name, picture, brief line that represents u, pronous -->
+    <div class="splash">
+      <h1>Leticia Silva Tran</h1>
+    </div>
+  </section>
+
+  <section class="about">
+    <!-- TODO: present yourself, some of your story, who u are, what u like, why.. goals-->
+    <p>Cupcake ipsum dolor sit amet sweet roll jelly. Jujubes soufflé sweet icing icing oat cake. Chocolate cake dragée chocolate bar. Candy tiramisu ice cream sugar plum. Sweet dessert gingerbread powder. Oat cake cheesecake macaroon wafer muffin cake gummies dessert jelly. Lollipop marshmallow icing pudding powder jelly beans.</p>
+    <!-- favorite quotes? -->
+  </section>
+
+
+  <section class="work-done">
+    <!-- TODO: brief explanation of code done and links to specific github repos -->
+    <section class="work-intro">
+      <h2>Sometitle Ada work...</h2>
+      <!-- <sub>This work... Ada Developers.... course.. full stack web develop...</sub> -->
+      <img src="https://placeimg.com/1200/200/any" alt="picture of Leticia S. Tran" />
+
+    </section>
+
+    <div class="git-startly">
```

Comment:
The rollover effects on these are neat, but they make me think I can click on them.